apisix/admin/init.lua

```diff
@@ -142,14 +162,8 @@ end
 
 
 local function run()
-    local api_ctx = {}
-    core.ctx.set_vars_meta(api_ctx)
-    ngx.ctx.api_ctx = api_ctx
-
-    local ok, err = check_token(api_ctx)
-    if not ok then
-        core.log.warn("failed to check token: ", err)
-        core.response.exit(401, {error_msg = "failed to check token"})
+    if not set_ctx_and_check_token() then
```

Comment:
no return value is needed, the `core.response.exit` will terminate the processing of the current request

Reply:
Thank you for pointing this out. I've updated the function to remove the value-returning logic, the function-call statements were updated as well.

PS: After this modification, I'm thinking that if "set_ctx_and_check_token" is still a good name, because the name didn't imply that the request would exit. Maybe "exit_if_token_invalid" is a better name?